core/build.gradle.kts

```diff
@@ -63,4 +71,17 @@ dependencies {
   implementation(Libs.kotlinx_coroutines_android)
   implementation(Libs.kotlinx_coroutines_rx3)
   implementation(Libs.zxing)
+
+  implementation(Libs.androidx_compose_material3)
+  implementation(Libs.androidx_activity_compose)
+
+  implementation(Libs.androidx_compose_ui)
```

Comment:
Since we have added ` androidx.compose.ui:ui-android:1.7.7` in our project. Due to this, we are facing a `PrivateResouce` lint issue for our `close_drawer` string. Can you please fix that? IMO add this in in lintConfig file since our app can have the same resource name, and we are getting this name from our `R` package so it will not conflict with compose internal string.

```
<issue id="PrivateResource" severity="warning" />
```

![image](https://github.com/user-attachments/assets/49079153-a944-412a-924e-06a4a6ec922b)